ambry-server/src/main/java/com/github/ambry/server/AmbryServerRequests.java

```diff
@@ -778,7 +778,6 @@ private ServerErrorCode handleRemoveStoreRequest(PartitionId partitionId) throws
     Store store = ((StorageManager) storeManager).getStore(partitionId, true);
     // Attempt to remove store from storage manager.
     if (storeManager.removeBlobStore(partitionId) && store != null) {
-      ((BlobStore) store).deleteStoreFiles();
```

Comment:
QQ, I saw that you move this into the removeBlobStore in StorageManager. Would it affect the CloudStorageManager?

Reply:
I think it might be okay since we don't have any local files deletion logic in the CloudStorageManager.